src/Bicep.LangServer/SemanticTokenVisitor.cs

```diff
@@ -351,5 +351,12 @@ public override void VisitUsingDeclarationSyntax(UsingDeclarationSyntax syntax)
             base.VisitUsingDeclarationSyntax(syntax);
         }
 
+        public override void VisitAssertDeclarationSyntax(AssertDeclarationSyntax syntax) 
```

Comment:
>VisitAssertDeclarationSyntax [](http://example.com/codeflow?start=29&length=28)

Btw, there's a duplicate visitor in the Wasm project that is used for the Bicep playground. Can you also add the same logic there?